src/frontend/js/components/PaymentButton/index.tsx

```diff
@@ -153,10 +154,12 @@ const PaymentButton = ({ billingAddress, creditCard, onSuccess }: PaymentButtonP
       let paymentInfos = payment;
 
       if (!paymentInfos) {
+        const cleanedBillingAddress = ObjectHelper.omit(billingAddress!, 'id', 'is_main');
```

Comment:
Suggestion about naming: `billingAddressPayload` or `billingAddressData`, the term `cleaned` does not make me think of the content of an API request instantly. It's up to you 😊